packages/devtools_app/lib/src/memory/memory_protocol.dart

```diff
@@ -5,19 +5,27 @@
 import 'dart:async';
 import 'dart:math' as math;
 
+import 'package:json_annotation/json_annotation.dart';
```

Comment:
We discussed offline that removing this json dependency might fix the build problem you were having.

Avoiding the extra json dependency will also make rolls into google3 easier.

Reply:
Done.